utils/tweet_search.py

```diff
@@ -130,4 +203,29 @@ def search_tweets(self, query: str,
 				print("Warning:", e)
 				print("Sleeping for 0.1 seconds and retrying")
 				retries += 1
-				time.sleep(0.1)
\ No newline at end of file
+				time.sleep(0.1)
+
+	@staticmethod
+	def __fix_floats(df: pd.DataFrame) -> pd.DataFrame:
+		#Select all float columns and Float64 columns (coerced after concatenating nullable values)
+		float_cols = df.select_dtypes(include=[float, "Float64"])
+
+		# Convert float columns to nullable Int64
+		for col in float_cols:
+			df[col] = df[col].astype('Int64')
+
+		return df
+
+	@staticmethod
```

Comment:
isn't this duplicated from the user code (and probably the search as well)?

Reply:
fix_floats removed